### DISCUSSION TITLE
Increase Hit Region on Features

### DISCUSSION THREAD

**Discussion by jamesmontemagno** [Mapsui/Mapsui#1773] [Q&A] [answered]
Right now I have a map with `LineString` using a `VectorStyle` to draw lines and I have a few `PintFeature` with `SymbolStyle` or `LabelStyle`.

I am processing and showing features based on:
```csharp
mapView.Info += MapView_Info;
```

This works, however you seem to need to be very precise. Is there a way to increase the hit area of the features without having to make the lines drawn or dots really big? I have already increased the base width to 6 and any bigger will make the map not look great at certain zoomes.

**Comment by charlenni** [comment, marked as answer]
No, there is up to now no good workaround for this. Mapsui checks for the clicked pixel with no enhancement. See [here](https://github.com/Mapsui/Mapsui/blob/b17b63db4a75b56c83602a915f235584b8c0934b/Mapsui.Rendering.Skia/MapRenderer.cs#L197).

**Reply by jamesmontemagno** [reply]
Thanks! I think it works fairly well, just a little padding in there would be nice. Much appreciate the code link.
